README.md

```diff
@@ -1,12 +1,14 @@
 # ADaaS Library
 
-Typescript ADaaS Library (@devrev/ts-adaas) provides:
+## Release Notes
 
-- type definitions for ADaaS control protocol,
-- an adapter for ADaaS control protocol,
-- helpers for uploading artifacts and manage the state for ADaaS snap-in.
+#### v1.0.0
 
-## Release Notes
+- Architecture overhaul with Node worker threads
+- Simplified metadata and data extraction with repos implementation
+- Handling of attachments streaming in library
+- Reduced file size, streamlined process by gzip compression
+- Bug fixes and improvements in error handling
 
```

Comment:
- Allow extractions to use full lambda runtime and gracefully handle execution context timeout.
- Simplified metadata and data normalization and uploading with repo implementation.
- Default handling of attachment extraction phase in ADaaS SDK library.
- Reduced file size, streamlined process by gzip compression.
- Bug fixes and improvements in error handling.